### ISSUE TITLE
Update the NWI spatial data - High Priority

### ISSUE THREAD

**Issue by PeteGoodwinUGS** [UGS-GIO/wetlands#76]
### Name of Existing Layer (as it appears in the application)

T_Wetlands; UT_Wetlands_Metadata; UT_Wetlands_nonRiverine; UT_Wetlands_Riverine; UT_Riparian_Metadata; UT_Riparian; LLWW Descriptors; LLWW Mapping Areas

### Folder path of data

U:\GWP\Wetland\websiteAndApps\mapApplication\most_recent_data

### Why do you feel this application needs what you're requesting?

The National Wetlands Inventory made a recent update to their data and our data needs to reflect the most up-to-date version of the NWI data. The new NWI data should completely replace the outdated NWI data on the web app.

We also completed another LLWW project and need to update the LLWW layers on the web map. The new LLWW data should be appended into the LLWW data on the web app.

### Who has reviewed your database so far (e.g. Schema, Field Types, Topology, etc.)?

No one has reviewed the NWI database, but I just replaced the old data with a new version of the data using the code Nate and Diane developed. 

The LLWW database has been formatted to match the database already on the web map.

### Who has reviewed your science and grammar so far?

Data have been reviewed by the USFWS.

**Comment by wetlandsDiane** [comment]
Should this issue be closed now?

**Comment by PeteGoodwinUGS** [comment]
Yep, it looks like the text was updated. I'll go ahead and close the issue.
Thanks for the reminder!
